.pre-commit-config.yaml

```diff
@@ -27,8 +27,8 @@ repos:
 #   cannot use --warn-unused-ignores because it conflicts with
 #     --ignore-missing-imports
 - repo: https://github.com/pre-commit/mirrors-mypy
-  rev: v1.1.1
+  rev: v0.930
```

Comment:
Why is this downgraded?

Reply:
Github actions runs nox, 0.930 is the version listed in `noxfile.py`. I am of the opinion they should be consistent.

Want me to upgrade the version nox uses instead?

Reply:
Done